parsons/bill_com/bill_com.py

```diff
@@ -101,6 +101,33 @@ def get_request_response(self, data, action, object_name, field="response_data")
         """
         return self.post_request(data, action, object_name)[field]
 
+    def paginate_list(self, response, data, object_name):
```

Comment:
I believe if you prepend the method with a `_` (e.g. `_paginate_list`) then it won't show up in the documentation. Otherwise, this internal method will. Alternatively, you can use `#` instead of `"""` and it won't show up in the documentation.

Looks like there are other methods in this class that have the same issue.